.eslintrc.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
     ]
   },
   parserOptions: {
-    ecmaVersion: 2020
+    ecmaVersion: 'latest'
```

Comment:
I'm not sure if we can do latest. We support 16+, so latest may have constructs that do not exist.

Reply:
CI would catch such cases. Node 16 is a lot more modern than folks give it credit for. It was released in April of 2021 (https://nodejs.org/en/blog/release/v16.0.0), well after V8 had implemented ECMAScript 2020.

Reply:
For what it's worth, Node 16 supports ECMAScript 2022 https://node.green/#ES2022